### PR TITLE
add document for system variable "tidb_enable_fast_table_check"

### PR DESCRIPTION
**Pull request by wjhuang2016**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1527,6 +1527,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 这个变量用来控制是否启用统计信息快速分析功能。默认值 0 表示不开启。
 - 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较低。这可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
 
+### `tidb_enable_fast_table_check`
```
Comment (qiancai) [inline system-variables.md:1530]:
```suggestion
### `tidb_enable_fast_table_check` <span class="version-mark">从 v7.2.0 版本开始引入</span>
```

File: system-variables.md [hunk 2]
```diff
@@ -1527,6 +1527,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+
+> **注意：**
+>
+> 该功能对多值索引和前缀索引不生效。
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 类型：布尔型
+- 默认值：`ON`
+- 这个变量用于控制是否使用基于校验和的方式来快速检查表数据和索引的一致性。默认值为 `ON`，即默认开启该功能。
```
Comment (qiancai) [inline system-variables.md:1540]:
```suggestion
- 这个变量用于控制是否使用基于校验和的方式来快速检查表中数据和索引的一致性。默认值 `ON` 表示该功能默认开启。
```

File: system-variables.md [hunk 3]
```diff
@@ -1527,6 +1527,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+- 开启后，TiDB 执行 `admin check table/index` 语句的速度更快。
```
Comment (qiancai) [inline system-variables.md:1541]:
```suggestion
- 开启后，TiDB 执行 [`ADMIN CHECK [TABLE|INDEX]`](/sql-statements/sql-statement-admin-check-table-index.md) 语句的速度更快。
```

File: system-variables.md [hunk 4]
```diff
@@ -1527,6 +1527,19 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 这个变量用来控制是否启用统计信息快速分析功能。默认值 0 表示不开启。
 - 快速分析功能开启后，TiDB 会随机采样约 10000 行的数据来构建统计信息。因此在数据分布不均匀或者数据量比较少的情况下，统计信息的准确度会比较低。这可能导致执行计划不优，比如选错索引。如果可以接受普通 `ANALYZE` 语句的执行时间，则推荐关闭快速分析功能。
 
+### `tidb_enable_fast_table_check`
+
+> **注意：**
+>
+> 该功能对多值索引和前缀索引不生效。
```
Comment (qiancai) [inline system-variables.md:1534]:
```suggestion
> 该功能对[多值索引](/sql-statements/sql-statement-create-index.md#多值索引)和前缀索引不生效。
```

**Review by tangenta — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**